x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/content.tsx

```diff
@@ -63,6 +70,14 @@ export function ContentPanel(props: ContentPanelProps) {
       );
     case 'policies':
       return <PackagePoliciesPanel name={name} version={version} />;
+    case 'custom':
+      return CustomView ? (
+        <ExtensionWrapper>
+          <CustomView />
```

Comment:
Should we pass some props to that custom view, like the name, version, ?

Reply:
Yeah, good point @nchaulet . I can see that information being useful to Integrations (maybe even the `lastestVersion` as well).

I will change the signature for the Component's props and add those in in a subsequent PR. Thanks for the feedback.